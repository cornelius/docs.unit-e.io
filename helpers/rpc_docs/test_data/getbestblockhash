getbestblockhash

Returns the hash of the best (tip) block in the longest blockchain.

Result:
"hex"      (string) the block hash hex encoded

Examples:
> unite-cli getbestblockhash
> curl --user myusername --data-binary someargs

