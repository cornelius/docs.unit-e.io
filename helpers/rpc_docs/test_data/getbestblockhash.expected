``getbestblockhash``

Returns the hash of the best (tip) block in the longest blockchain.

Result
~~~~~~

.. list-table::
   :header-rows: 1

   * - Name
     - Type
     - Description
   * - hex
     - string
     - the block hash hex encoded

Examples
~~~~~~~~

::

  unite-cli getbestblockhash

::

  curl --user myusername --data-binary someargs

